### DISCUSSION TITLE
Add a keyboard shortcut to changes tabs

### DISCUSSION THREAD

**Discussion by AlexandreBonneau** [freeplane/freeplane#1679] [Ideas]
It would be great to be able to change tabs using a keyboard shortcut similar to the one in spreadsheet editors, ie. Ctrl+pgup to select the previous tab, and Ctrl+pgdown to select the next tab.

**Comment by euu2021** [comment]
It exists. Assign a hotkey to:
`Navigate->Next map`
`Navigate->Previous map `

**Comment by AlexandreBonneau** [comment]
I just did and it works great, thanks.
Shouldn't those shortcuts be created by default though?

> EDIT: To add a keyboard shortcut, navigate to the menu, then hover the menu with your mouse and use Ctrl+left click to open the dialog to edit the keyboard shortcut

**Comment by herberteuler** [comment]
Second this. It would be great if there were default shortcuts for them.
